src/main/java/com/futurecollars/zad3/BaseEmployee.java

```diff
@@ -0,0 +1,27 @@
+package com.futurecollars.zad3;
+
+public abstract class BaseEmployee {
+    protected String name;
+    protected String lastname;
+    protected int baseSalary = 3000;
```

Comment:
daj mozliwosc ustawiania baseSalary w klasach podrzędnych -> ustawiaj to w konstruktorze